### PR TITLE
[Hub Generated] Review request for Microsoft.ContainerRegistry to add version preview/2023-01-01-preview

### PR DESCRIPTION
**Pull request by jikuma**
This is a PR generated at [OpenAPI Hub](https://aka.ms/openapihub). You can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/Azure/azure-rest-api-specs/jikuma-containerregistry-Microsoft.ContainerRegistry-2023-01-01-preview...main/).
## ARM API Information (Control Plane)

Azure 1st Party Service can try out the [Shift Left](https://aka.ms/ShiftLeft) experience to initiate API design review from ADO code repo.  If you are interested, may request engineering support by filling in with the form https://aka.ms/ShiftLeftSupportForm.

### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [ ] new service onboarding
      - [ ] new API version
      - [ ] update existing version for new feature
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify
  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month.
  3. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month.
  4. By default, Azure SDKs of all languages (.NET/Python/Java/JavaScript for both management-plane SDK and data-plane SDK, Go for management-plane SDK only ) MUST be refreshed with/after swagger of new version is published. If you prefer NOT to refresh any specific SDK language upon swagger updates in the current PR, please leave details with justification here.

### Contribution checklist (MS Employees Only):
- [x] I commit to follow the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy) of "no breaking changes"
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist

> **Applicability**: :warning:
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
> - Change to data plane APIs
> - Adding new properties
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [ ] Check this box if any of the following apply to the PR so that the label "ARMReview" and "WaitForARMFeedback" will be added by bot to kick off ARM API Review.  Missing to check this box in the following scenario may result in delays to the ARM manifest review and deployment.
  - Adding a new service
  - Adding new API(s)
  - Adding a new API version
    -[ ] To review changes efficiently, ensure you copy the existing version into the new directory structure for first commit and then push new changes, including version updates, in separate commits. You can use OpenAPIHub to initialize the PR for adding a new version. For more details refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).

- [ ] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them.

### Breaking Change Review Checklist
If you have any breaking changes as defined in the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy/), request approval from the Breaking Change Review Board.

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Additional details on the process and office hours are on the [Breaking Change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

NOTE: To update API(s) in public preview for over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json [hunk 1]
```diff
@@ -15,6 +15,527 @@
...
+          "default": {
+            "description": "Error response describing why the operation failed. If the resource does not exist, 404 (Not found) is returned.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ArchiveGet": {
+            "$ref": "./examples/ArchiveGet.json"
```
Comment (ms-zhenhua) [inline specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json:109]: >ArchiveGet [](http://example.com/codeflow?start=32&length=10)

Please check file name under `example`

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json [hunk 2]
```diff
@@ -15,6 +15,527 @@
...
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful; the operation has completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/Archive"
+            }
+          },
+          "201": {
+            "description": "The request was successful; the operation will complete asynchronously.",
```
Comment (ms-zhenhua) [inline specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json:156]: >"The request was successful; the operation will complete asynchronously." [](http://example.com/codeflow?start=27&length=73)

"ARM create operation completed successfully."

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json [hunk 3]
```diff
@@ -15,6 +15,527 @@
...
+            "description": "The parameters for creating a archive.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Archive"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful; the operation has completed successfully.",
```
Comment (ms-zhenhua) [inline specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json:150]: >"The request was successful; the operation has completed successfully." [](http://example.com/codeflow?start=27&length=71)

"ARM update operation completed successfully."

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json [hunk 4]
```diff
@@ -15,6 +15,527 @@
...
+            "headers": {
+              "Azure-AsyncOperation": {
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed. If any of the input parameters are invalid, 400 (Bad Request) is returned. If the request cannot be processed due to a conflict in the request, 409 (Conflict) is returned.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
```
Comment (ms-zhenhua) [inline specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json:169]: >"$ref": "#/definitions/ErrorResponse" [](http://example.com/codeflow?start=14&length=37)

use "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json [hunk 5]
```diff
@@ -15,6 +15,527 @@
...
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ArchiveCreate": {
+            "$ref": "./examples/ArchiveCreate.json"
```
Comment (ms-zhenhua) [inline specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json:175]: >ArchiveCreate [](http://example.com/codeflow?start=32&length=13)

Please check file name under example

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json [hunk 6]
```diff
@@ -15,6 +15,527 @@
...
+              }
+            }
+          },
+          "204": {
+            "description": "The archive does not exist in the subscription."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
```
Comment (ms-zhenhua) [inline specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json:227]: >"$ref": "#/definitions/ErrorResponse" [](http://example.com/codeflow?start=14&length=37)

use "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json [hunk 7]
```diff
@@ -15,6 +15,527 @@
...
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ArchiveDelete": {
+            "$ref": "./examples/ArchiveDelete.json"
```
Comment (ms-zhenhua) [inline specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json:233]: >ArchiveDelete [](http://example.com/codeflow?start=32&length=13)

Please check file name under example

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json [hunk 8]
```diff
@@ -15,6 +15,527 @@
...
+            "description": "The request was successful; the operation will complete asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/Archive"
+            },
+            "headers": {
+              "Azure-AsyncOperation": {
+                "type": "string"
+              }
+            }
+          },
```
Comment (ms-zhenhua) [inline specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json:293]: LRO patch should use 200 and 202:
[https://github.com/Azure/azure-openapi-validator/blob/main/docs/patch-response-code.md](https://github.com/Azure/azure-openapi-validator/blob/main/docs/patch-response-code.md)

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json [hunk 9]
```diff
@@ -15,6 +15,527 @@
...
+          "default": {
+            "description": "Error response describing why the operation failed. If any of the input parameters are invalid, 400 (Bad Request) is returned. If the request cannot be processed due to a conflict in the request, 409 (Conflict) is returned.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
```
Comment (ms-zhenhua) [inline specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json:297]: >"$ref": "#/definitions/ErrorResponse" [](http://example.com/codeflow?start=14&length=37)

use "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json [hunk 10]
```diff
@@ -15,6 +15,527 @@
...
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ArchiveUpdate": {
+            "$ref": "./examples/ArchiveUpdate.json"
```
Comment (ms-zhenhua) [inline specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json:303]: >ArchiveUpdate [](http://example.com/codeflow?start=32&length=13)

Please check file name under example

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json [hunk 11]
```diff
@@ -15,6 +15,527 @@
...
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ArchiveVersionList": {
+            "$ref": "./examples/ArchiveVersionList.json"
```
Comment (ms-zhenhua) [inline specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json:355]: >ArchiveVersionList [](http://example.com/codeflow?start=32&length=18)

Please check file name under example

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json [hunk 12]
```diff
@@ -15,6 +15,527 @@
...
+          "default": {
+            "description": "Error response describing why the operation failed. If the resource does not exist, 404 (Not found) is returned.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ArchiveVersionGet": {
+            "$ref": "./examples/ArchiveVersionGet.json"
```
Comment (ms-zhenhua) [inline specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json:409]: >ArchiveVersionGet [](http://example.com/codeflow?start=32&length=17)

Please check file name under example

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json [hunk 13]
```diff
@@ -15,6 +15,527 @@
...
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful; the operation has completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/ArchiveVersion"
+            }
+          },
+          "201": {
+            "description": "The request was successful; the operation will complete asynchronously.",
```
Comment (ms-zhenhua) [inline specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json:450]: >"The request was successful; the operation will complete asynchronously." [](http://example.com/codeflow?start=27&length=73)

"ARM create operation completed successfully."

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json [hunk 14]
```diff
@@ -15,6 +15,527 @@
...
+          {
+            "$ref": "#/parameters/ArchiveNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ArchiveVersionNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful; the operation has completed successfully.",
```
Comment (ms-zhenhua) [inline specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json:444]: >"The request was successful; the operation has completed successfully." [](http://example.com/codeflow?start=27&length=71)

"ARM update operation completed successfully."

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json [hunk 15]
```diff
@@ -15,6 +15,527 @@
...
+            "headers": {
+              "Azure-AsyncOperation": {
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed. If any of the input parameters are invalid, 400 (Bad Request) is returned. If the request cannot be processed due to a conflict in the request, 409 (Conflict) is returned.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
```
Comment (ms-zhenhua) [inline specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json:463]: >"$ref": "#/definitions/ErrorResponse" [](http://example.com/codeflow?start=14&length=37)

use "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json [hunk 16]
```diff
@@ -15,6 +15,527 @@
...
+              }
+            }
+          },
+          "204": {
+            "description": "The archive does not exist in the subscription."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
```
Comment (ms-zhenhua) [inline specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json:524]: >"$ref": "#/definitions/ErrorResponse" [](http://example.com/codeflow?start=14&length=37)

use "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json [hunk 17]
```diff
@@ -15,6 +15,527 @@
...
+          "default": {
+            "description": "Error response describing why the operation failed. If any of the input parameters are invalid, 400 (Bad Request) is returned. If the request cannot be processed due to a conflict in the request, 409 (Conflict) is returned.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ArchiveVersionCreate": {
+            "$ref": "./examples/ArchiveVersionCreate.json"
```
Comment (ms-zhenhua) [inline specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json:469]: >ArchiveVersionCreate [](http://example.com/codeflow?start=32&length=20)

Please check file name under example

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json [hunk 18]
```diff
@@ -15,6 +15,527 @@
...
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ArchiveVersionDelete": {
+            "$ref": "./examples/ArchiveVersionDelete.json"
```
Comment (ms-zhenhua) [inline specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json:530]: >ArchiveVersionDelete [](http://example.com/codeflow?start=32&length=20)

Please check file name under example

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json [hunk 19]
```diff
@@ -15,6 +15,527 @@
     "application/json"
   ],
   "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerRegistry/registries/{registryName}/packages/{packageType}/archives": {
+      "get": {
+        "tags": [
+          "Archives"
+        ],
+        "description": "Lists all archives for the specified container registry and repository type.",
+        "operationId": "Archives_List",
```
Comment (ms-zhenhua) [inline specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json:24]: >Archives_List [](http://example.com/codeflow?start=24&length=13)

Archives_ListByPackageType

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json [hunk 20]
```diff
@@ -15,6 +15,527 @@
     "application/json"
   ],
   "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerRegistry/registries/{registryName}/packages/{packageType}/archives": {
+      "get": {
+        "tags": [
+          "Archives"
+        ],
+        "description": "Lists all archives for the specified container registry and repository type.",
```
Comment (ms-zhenhua) [inline specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json:23]: >repository [](http://example.com/codeflow?start=84&length=10)

repository or package ?

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json [hunk 21]
```diff
@@ -15,6 +15,527 @@
...
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerRegistry/registries/{registryName}/packages/{packageType}/archives/{archiveName}/versions": {
+      "get": {
+        "tags": [
+          "ArchiveVersions"
+        ],
+        "description": "Lists all archive versions for the specified container registry, repository type and archive name.",
+        "operationId": "ArchiveVersions_List",
```
Comment (ms-zhenhua) [inline specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json:318]: >ArchiveVersions_List [](http://example.com/codeflow?start=24&length=20)

ArchiveVersions_ListByArchiveName

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json [hunk 22]
```diff
@@ -15,6 +15,527 @@
...
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerRegistry/registries/{registryName}/packages/{packageType}/archives/{archiveName}/versions/{archiveVersionName}": {
+      "get": {
+        "tags": [
+          "ArchiveVersions"
+        ],
+        "description": "Gets the properties of the archive version.",
+        "operationId": "ArchiveVersions_Get",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v4/types.json#/parameters/ApiVersionParameter"
```
Comment (ms-zhenhua) [inline specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json:372]: >v4 [](http://example.com/codeflow?start=69&length=2)

please use the latest version "v5" please also update other places

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json [hunk 23]
```diff
@@ -3637,6 +4158,169 @@
...
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "ArchivePackageSourceProperties": {
+      "description": "The properties of the archive package source.",
+      "type": "object",
+      "properties": {
+        "type": {
```
Comment (ms-zhenhua) [inline specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json:4202]: >type [](http://example.com/codeflow?start=9&length=4)

is it enumerable ?

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json [hunk 24]
```diff
@@ -3637,6 +4158,169 @@
...
+          "description": "The properties of the archive.",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "ArchiveVersionProperties": {
+      "description": "The properties of an export pipeline.",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
```
Comment (ms-zhenhua) [inline specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json:4270]: >provisioningState [](http://example.com/codeflow?start=9&length=17)

duplicated with that in "ArchiveProperties". please consider define a separate "ProvisioningState" definition

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json [hunk 25]
```diff
@@ -7248,6 +7932,39 @@
       "type": "string",
       "x-ms-parameter-location": "method"
     },
+    "PackageTypeParameter": {
+      "name": "packageType",
```
Comment (ms-zhenhua) [inline specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json:7938]: >packageType [](http://example.com/codeflow?start=15&length=11)

is it enumerable ?

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json (hunk 3 above)
Comment (jikuma) [reply specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json:150]: The description of the response code 200 is same for all the resources.

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json (hunk 2 above)
Comment (jikuma) [reply specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json:156]: The description of the response code 200 is same for all the resources.

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json (hunk 4 above)
Comment (jikuma) [reply specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json:169]: Its same for other resources too. I did not do any custom changes for new Archive Resource.

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json (hunk 6 above)
Comment (jikuma) [reply specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json:227]: It's similar to other resources.

**Comment by keryul** [comment]
@jikuma can you fix the BreakingChange and PrettierCheck error?

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json (hunk 4 above)
Comment (zizw123) [reply specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json:169]: You should use the ErrorResponse which will fix the lint error, if you are adding new resource and new API, it is a good opportunity to adopt the right behavior.

Comment (zizw123) [reply specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json:169]: > > "$ref": "#/definitions/ErrorResponse"
> 
> use "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"

See this response, the only time you should be suppressing the lint error is when it's false alarm

**Pull request closed without merging**